test/elements.test.ts

```diff
@@ -0,0 +1,37 @@
+import { ElementDictionary } from "../src/data/elements";
+import { Phases, Phase } from "../src/interfaces/element-props";
+
+const elements: string[] = Object.keys(ElementDictionary);
```

Comment:
Why is this global variable defined? You could define it inside the function where it is used instead.

Reply:
To contain , element names

Reply:
But it need not be a global variable. Can You can declare the variable inside the test block instead?

Reply:
@qur786 please address this change too. Keep it inside describe block if other test cases are using it too.